amazon-eks-nodegroup.yaml

```diff
@@ -155,6 +175,10 @@ Metadata:
           - NodeGroupName
           - NodeAutoScalingGroupMinSize
           - NodeAutoScalingGroupMaxSize
+          - NodeAutoScalingGroupDesiredCapacity
+          - NodeAutoScalingGroupUpdatePolicyMaxBatchSize
+          - NodeAutoScalingGroupUpdatePolicyMinInstancesInService
+          - NodeAutoScalingGroupUpdatePolicyPauseTime
```

Comment:
This is getting a little dense, any opinions about moving the UpdatePolicy parameters to their own ParameterGroup for UI cleanliness?

Reply:
Maybe all 6 ASG settings in that case.